pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -70,6 +70,9 @@ type DependenciesDistributor struct {
 	RESTMapper          meta.RESTMapper
 	ResourceInterpreter resourceinterpreter.ResourceInterpreter
 
+	// ConcurrentDistributorSyncs is the number of bindings that are allowed to sync concurrently.
+	ConcurrentDistributorSyncs int
```

Comment:
Suggest renaming it to `ConcurrentDependenciesDistributorSyncs`, this shows tath it is set up as dependent distribution.

Reply:
updated, please have a look.